hudi-common/src/main/java/org/apache/hudi/common/table/view/RemoteHoodieTableFileSystemView.java

```diff
@@ -176,7 +178,8 @@ private <T> T executeRequest(String requestPath, Map<String, String> queryParame
         response = Request.Post(url).connectTimeout(timeout).socketTimeout(timeout).execute();
         break;
     }
-    String content = response.returnContent().asString();
+    Charset charset = Consts.UTF_8;
+    String content = response.returnContent().asString(charset);
```

Comment:
@nsivabalan we have the support of UTF-8 in the base path so this should be fine?

Reply:
Yes, it is found that only the append mode in Spark has this problem.